core/org.wso2.carbon.registry.core/src/main/java/org/wso2/carbon/registry/core/jdbc/realm/RegistryAuthorizationManager.java

```diff
@@ -369,6 +369,21 @@ public void clearRoleActionOnAllResources(String roleName, String action)
         return getAuthorizationManager().getAllowedUIResourcesForUser(userName, permissionRootPath);
     }
 
+    /**
+     * Get the allowed UI resources for a role.
+     *
+     * @param roleName
+     *            the user name.
```

Comment:
permissionRootPath is not specified in the comment. Comment need to be formatted.